renderer/components/ActivitiesSection.tsx

```diff
@@ -90,8 +131,25 @@ export default function ActivitiesSection({
           activities={activities}
           onDeleteActivity={onDeleteActivity}
           selectedDate={selectedDate}
+          setShowGoogleEvents={setShowGoogleEvents}
+          formattedGoogleEvents={
+            showGoogleEvents &&
+            googleEvents.length > 0 &&
+            formattedGoogleEvents.length > 0
+              ? formattedGoogleEvents
+              : undefined
+          }
         />
       </div>
+
+      <div className="flex gap-2 px-6 pb-4 items-center justify-end mr-auto">
+        {today && isLogged && (
+          <GoogleCalendarShowCheckbox
+            setShowGoogleEvents={setShowGoogleEvents}
+          />
+        )}
+      </div>
+
```

Comment:
Checkbox is not synchronized with events
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/133204916/e87678f5-c273-4da3-a014-4126feea511b)

Reply:
And it would be nice to be able to show Google events without having any events today
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/133204916/f2297cbe-69a1-42b5-a8d3-4f93e94b736a)